src/main/java/ua/javarush/caesarcipherapp/dao/FileAccess.java

```diff
@@ -0,0 +1,71 @@
+package ua.javarush.caesarcipherapp.dao;
+
+
+import ua.javarush.caesarcipherapp.view.ConsoleViewProvider;
+
+import java.io.*;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.util.ArrayList;
+import java.util.List;
+
+public class FileAccess {
+
+    private final String incomingPath;
+
+    ConsoleViewProvider consoleViewProvider = new ConsoleViewProvider();
+
+    public FileAccess(String incomingPath) {
+        this.incomingPath = incomingPath;
+    }
+
+    public List<Character> dataReading() {
+        List<Character> allChars = new ArrayList<>();
+        try (BufferedReader bfReader = new BufferedReader(new FileReader(incomingPath))) {
+            while (bfReader.ready()) {
+                allChars.add((char) bfReader.read());
+            }
+        } catch (IOException e) {
+            throw new IllegalArgumentException(e);
+        }
+        return allChars;
+    }
+
+    public void fileCreating(List<Character> list, String name) {
```

Comment:
Could please specify what these parameters mean list and name in the context of the current method?